components/builder-depot/src/config.rs

```diff
@@ -52,6 +52,8 @@ pub struct Config {
     /// Upstream depot to pull packages from if someone tries to install from this depot and they
     /// aren't present. This is optional because e.g. public Builder doesn't have an upstream.
     pub upstream_depot: Option<String>,
+    // Origins for which we pull from upstream (default: core)
+    pub upstream_origins: String,
```

Comment:
Why a `String` instead of a `Vec<String>`?

Reply:
Ah good point - I have it set up to take a comma separated string with origins, but having it be a Vec at the config level would be better - I'll update it.